documentation/package.json

```diff
@@ -12,7 +12,9 @@
   "author": "DESCW",
   "license": "ISC",
   "devDependencies": {
-    "@vuepress/plugin-search": "^2.0.0-rc",
-    "vuepress": "^2.0.0-rc"
+    "@vuepress/plugin-search": "^2.0.0-rc.0",
+    "@vuepress/theme-default": "^2.0.0-rc.0",
+    "@vuepress/bundler-vite": "^2.0.0-rc.0",
+    "vuepress": "^2.0.0-rc.0"
```

Comment:
was there a reason you didn't use https://www.npmjs.com/package/vuepress/v/2.0.0-rc.2

Reply:
so i did, i guess it changed in that span, next time i probably shouldn't specify :(